docs/basics/1d_hist.rst

```diff
@@ -265,3 +265,18 @@ String category
 .. image:: ../img/1d_str_category.svg
     :alt: String category plot
     :width: 500
+
+
+.. _basics-1d_hist_side_by_side-label:
+Using multiple histograms
+-------------------------
+
+With multiple histograms, the :func:`plot_hist() <plothist.plotters.plot_hist>` function will correctly put them side by side:
```

Comment:
because it is a wrapper around the `hist()` function matplotlib that provides this functionality.